.coveragerc

```diff
@@ -12,4 +12,7 @@ omit =
     #src/RestrictedPython/tests/*.py
 
 [report]
-precision = 3
+precision = 2
+
+[html]
+directory = reports/coverage
```

Comment:
well the flake8-html requires that this directory "reports" exists, but 

    directory = reports/coverage 

is just for coverage reports whcih will be created there then. This meens that all reprots are then created under a reports directiory,